style.css

```diff
@@ -0,0 +1,77 @@
+blockquote {
+  max-width: 550px;
+  margin: 0 10px;
+  font-style: italic;
+  text-align: left;
+}
+
+html {
+  font-size: 10px;
+  display: block;
+}
+
+body {
+  display: block;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    "Roboto",
+    "Helvetica Neue",
+    Arial,
+    sans-serif;
+  font-size: 1.6rem;
+  line-height: 1.5;
+  text-align: center;
+  color: #333;
+  margin: 0;
+}
+
+#main {
+  margin: 30px 8px;
+  padding: 15px;
+  border-radius: 5px;
+  background: #eee;
+}
+
+.tribute-info-itle {
+  padding: 20px 0 30px 0;
```

Comment:
great job. Please for padding you can two values that will work same as using 4 values

Reply:
Hello @alicemirigo92, please take into account that this request isn't very clear 😉. Remember to always double-check your English before sending a review, as this will give the student a better time understanding what should be updated. Also, try to make explicit what should be changed and _why_ it should be changed 😄! Consider using [Grammarly](https://www.grammarly.com/) to check your sentences.